### ISSUE TITLE
Irritations in new LTS table

### ISSUE THREAD

**Issue by froschdesign** [zendframework/zf3-web#103]
## Problem

zend-stratigility 3.0.0 was released on [15 Mar 2018](https://github.com/zendframework/zend-stratigility/releases/tag/3.0.0) and the previous minor release (2.2) enters its security support phase, but this does not mean that the entire package receives only security support.

The table says something else:

| Package                             | Status   | Supported Version(s) | Support Ends |
|-------------------------------------|----------|----------------------|--------------|
| zendframework/zend-di               | SECURITY | 3.0, 2.6             | 2018-11-30   |
| zendframework/zend-permissions-rbac | SECURITY | 3.0, 2.6             | 2019-03-22   |
| zendframework/zend-stratigility     | SECURITY | 3.0, 2.2             | 2019-03-15   |

The security status and the support end is correct for the lower versions of these packages, but not for version 3.0.

## Suggestion

Extend the table like:

| Package                             | Status   | Supported Version(s) | Support Ends |
|-------------------------------------|----------|----------------------|--------------|
| zendframework/zend-di               | ACTIVE, SECURITY | 3.0, 2.6             | N/A, 2018-11-30   |
| zendframework/zend-permissions-rbac | ACTIVE, SECURITY | 3.0, 2.6             | N/A, 2019-03-22   |
| zendframework/zend-stratigility     | ACTIVE, SECURITY | 3.0, 2.2             | N/A, 2019-03-15   |

or rearrange the table:

| Package                             | Status   | Supported Version(s) | Support Ends |
|-------------------------------------|----------|----------------------|--------------|
| zendframework/zend-di               | ACTIVE | 3.0             | N/A   |
|                                                        | SECURITY | 2.6             | 2018-11-30   |
| zendframework/zend-permissions-rbac | ACTIVE | 3.0             | N/A   |
|              | SECURITY | 2.6             | 2019-03-22   |
| zendframework/zend-stratigility     | ACTIVE | 3.0             | N/A   |
|      | SECURITY | 2.2             | 2019-03-15   |

**Comment by froschdesign** [comment]
@weierophinney 
The newsletter is out, but the problem still exists.

**Comment by weierophinney** [comment]
Yes, totally aware, and know what changes I need to make; I hope to have them up and tested no later than Monday.

**Comment by weierophinney** [comment]
So, #104 MOSTLY fixed things, but I had some production issues (in particular, skeletons related to an LTS were not displaying; also there was a bug in the code that did the actual fetch from GitHub). Additionally, I noted that packages were not sorted, which made finding relevant information for a component really, really difficult.

I've fixed all of these issues as of today, however, and the table now accurately reflects reality!
